### PR TITLE
Added Button compoment.

### PR DESCRIPTION
**Pull request by sebascomeau**

### CODE REVIEW

File: scv-prototype-frontend/src/app/components/Button/index.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	const disabledProp = disabled ? { disabled: "disabled" } : undefined;
+
+	return <button type={type}
+		className={
+			"btn" +
+			` btn-${variant}` +
+			(size && size !== Button.sizes.default ? ` btn-${size}` : "") +
+			(className ? ` ${className}` : "")
+		}
+		{...disabledProp}
```
Comment (gregory-j-baker) [inline scv-prototype-frontend/src/app/components/Button/index.jsx]: Is this necessary? React is usually pretty smart when it comes to this stuff. I mean, you can use `disabled={disabled}` and if the value is undefined, it won't render it. (**I think**)

File: scv-prototype-frontend/src/app/views/Login/Login.jsx [hunk 2]
```diff
@@ -15,9 +16,9 @@ const Login = (props) => {
 	const [password, setPassword] = useState('password');
 	const [rememberMe, setRememberMe] = useState();
 
-	const {t} = useTranslation();
-	const {error, loading, login} = useApi();
-	const {authenticationContext} = {...useContext(AuthenticationContext)};
+	const { t } = useTranslation();
+	const { error, loading, login } = useApi();
+	const { authenticationContext } = { ...useContext(AuthenticationContext) };
```
Comment (gregory-j-baker) [inline scv-prototype-frontend/src/app/views/Login/Login.jsx]: Are we using `{ foo }` or `{foo}` here. The react style guide doesn't specifically mention decomposition; it mentions using `{...}` _in react props_.

Comment (sebascomeau) [reply scv-prototype-frontend/src/app/views/Login/Login.jsx]: The spacings were added by VS Code when I saved the file.

File: scv-prototype-frontend/src/app/components/Button/index.jsx (hunk 1 above)
Comment (sebascomeau) [reply scv-prototype-frontend/src/app/components/Button/index.jsx]: I was not sure how to implement it. I will try your approach.

**Pull request merged**